src/content/docs/it/guides/upgrade-to/v2.mdx

```diff
@@ -0,0 +1,423 @@
+
+- `markdown.gfm: false` disables GitHub-Flavored Markdown
+- `markdown.smartypants: false` disables SmartyPants
+
+```js del={6} ins={7-8}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+
+export default defineConfig({
+  markdown: {
+    extendDefaultPlugins: false,
+    smartypants: false,
+    gfm: false,
+  }
+});
+```
+
+### Cambiato: Configurazione del plugin MDX
+
+#### Sostituito: `extendPlugins` cambiato con `extendMarkdownConfig`
+
+Nella v1.x, l'opzione `extendPlugins` dell'integrazione MDX gestiva come i tuoi file MDX dovevano ereditare le configurazioni di Markdown: tutte le tue configurazioni di Markdown (`markdown`), o solamente i plugin predefiniti di Astro (`default`).
+
+Astro v2.0 sostituisce questo comportamento controllato da `mdx.extendPlugins` con tre nuove opzioni, configurabili indipendentemente, che sono impostate a `true`:
+
+- **[`mdx.extendMarkdownConfig`](/it/guides/integrations-guide/mdx/#extendmarkdownconfig)** to inherit all or none of your Markdown configuration
+- **`mdx.gfm`** to enable or disable GitHub-Flavored Markdown in MDX
+- **`mdx.smartypants`** to enable or disable SmartyPants in MDX
+
+##### Cosa devo fare?
+
+Elimina `extendPlugins: 'markdown'` nella tua configurazione. Ora questo comportamento è predefinito.
+
+```js del={8}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  integrations: [
+    mdx({
+      extendPlugins: 'markdown',
+    }),
+  ],
+});
+```
+
+Sostituisci `extendPlugins: 'defaults'` con `extendMarkdownConfig: false` e aggiungi le opzioni separate per GitHub-Flavored Markdown e SmartyPants per abilitare indipendentemente questi plugin predefiniti in MDX.
+
+```js del={8} ins={9-11}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  integrations: [
+    mdx({
+      extendPlugins: 'defaults',
+      extendMarkdownConfig: false,
+      smartypants: true,
+      gfm: true,
+    }),
+  ],
+});
+```
+
+#### Aggiunto: Maggiori opzioni di configurazione di MDX per stare al passo con Markdown
+
+Astro v2.0 ti permette ora di impostare individualmente [ogni possibile opzione di configurazione di Markdown](/it/reference/configuration-reference/#markdown-options) (eccetto `drafts`) separatamente nella configurazione della tua integrazione MDX.
+
+```js
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import mdx from '@astrojs/mdx';
+
+export default defineConfig({
+  markdown: {
+    remarkPlugins: [remarkPlugin1],
+    gfm: true,
+  },
+  integrations: [
+    mdx({
+      remarkPlugins: [remarkPlugin2],
+      gfm: false,
+    })
+  ]
+});
+```
+
+##### Cosa devo fare?
+
+Rivedi le configurazioni di Markdown e MDX e confronta la tua configurazione con le nuove opzioni disponibili.
+
+
+### Cambiato: Accesso dei plugn all'avantesto
```

Comment:
**issue:** I think the word `avantesto` is out of context here (see e.g. https://www.treccani.it/vocabolario/avantesto/).
I would keep the english work `frontmatter` which is well understood in the context of markdown files.